src/pages/settings/Payments/PaymentMethodList.js

```diff
@@ -214,36 +219,33 @@ class PaymentMethodList extends Component {
     render() {
         return (
             <>
-                <FlatList
-                    data={this.getFilteredPaymentMethods()}
-                    renderItem={this.renderItem}
-                    keyExtractor={item => item.key}
-                    ListEmptyComponent={this.renderListEmptyComponent()}
-                    ListHeaderComponent={this.props.listHeaderComponent}
-                />
-                {
-                    this.props.shouldShowAddPaymentMethodButton
-                    && (
-                        <FormAlertWrapper>
```

Comment:
Why did you get rid of the form alert wrapper? Is it sort of a deprecated component? I guess it's fine since to remove it since we were only using the `isOffline` prop.

Reply:
Its not deprecated, but we are already pulling the network. And the formalertwrapper component essentially just does that

Reply:
So I figure just do offline stuff here and all at once